htlcswitch/link.go

```diff
@@ -952,7 +952,7 @@ func (l *channelLink) htlcManager() {
 
 			l.fail(
 				LinkFailureError{
-					code:       ErrSyncError,
+					code:       ErrRecoveryError,
```

Comment:
Safe change? I know some implementations string match on the previous error message...

Reply:
In this case it is, as this version won't timeout on chan reest anymore. Also see my comment above w.r.t how that change by other implementations actually caused them to ignore our DLP recovery at times, since they wouldn't close based on that error.